packages/react-form/src/hooks/visitFields.ts

```diff
@@ -10,9 +10,12 @@ export default function useVisitFields(
   fieldBag: {[key: string]: FieldOutput<any>},
   visitor: FieldVisitor,
 ) {
+  const fieldBagRef = useRef(fieldBag);
+  fieldBagRef.current = fieldBag;
```

Comment:
Could be a useful generic hook?

Reply:
true, though I do like that if you are using actual `useRef` the linting rule knows not to force you to add it to deps arrays